package.json

```diff
@@ -86,7 +86,7 @@
 		"dev:test-env": "pnpm --filter ./sites/test-env dev",
 		"build:docs": "pnpm --filter ./sites/docs build",
 		"build:example-project": "pnpm --filter ./sites/example-project build",
-		"build:production": "VITE_BUILD_STRICT=true pnpm --filter ./sites/example-project build",
```

Comment:
is this a breaking change?

Reply:
Only within the monorepo, everything that is end-user facing lives behind the `build:strict` cli command, which is still supported for now